### PR TITLE
[Term Entry] Go Math Function: Floor()

### PR DESCRIPTION
**Pull request by frhadjmli**
### Description

- Add `math.Floor()`  term for Go
- Closes issue #2439 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/floor/floor.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+Title: 'Floor()'
+Description: 'Returns the given decimal number rounded down to the nearest lower value.'
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/floor/floor.md:3]:
```suggestion
Description: 'Returns the given decimal number rounded down to the nearest whole number.'
```

File: content/go/concepts/math-functions/terms/floor/floor.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Floor()`** function returns a given decimal number rounded down to the nearest lower value.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/floor/floor.md:16]:
```suggestion
The **`Floor()`** function returns a given decimal number rounded down to the nearest whole number.
```

File: content/go/concepts/math-functions/terms/floor/floor.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+## Syntax
+
+```pseudo
+math.Floor(decimalNumber float64) float64
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/floor/floor.md:21]:
```suggestion
math.Floor(decimalNumber) 
```

File: content/go/concepts/math-functions/terms/floor/floor.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+```
+
+The `Floor()` function accepts a single parameter of type float64 and rounds it down to the nearest lower value.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/floor/floor.md:24]:
```suggestion
The `Floor()` function accepts a single parameter of type `float64` and rounds it down to the nearest whole number.
```

File: content/go/concepts/math-functions/terms/floor/floor.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+Take notice that the returned value may appear to be an `int`, but it is a `float64` value.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/floor/floor.md:25]:
```suggestion

> **Note:** The returned value may appear to be an `int`, but it is a `float64` value.
```

File: content/go/concepts/math-functions/terms/floor/floor.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+In some exceptional cases, passing any of the following values: `±Inf`, `±0`, and `NaN` as an argument to the `Floor()` function, will result in the returned value being the same as the argument.
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/floor/floor.md:27]:
```suggestion
> **Note:** Passing any of the following values: `±Inf`, `±0`, and `NaN` as an argument to the `Floor()` function, will result in the returned value being the same as the argument.
```

File: content/go/concepts/math-functions/terms/floor/floor.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+## Example
+
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/floor/floor.md:30]: There should be a short introductory sentence here

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@frhadjmli I've added my comments. Please update the entry, and tag me when you've finished making changes. Thanks

File: content/go/concepts/math-functions/terms/floor/floor.md (hunk 7 above)
Comment (frhadjmli) [reply content/go/concepts/math-functions/terms/floor/floor.md:30]: @caupolicandiaz You mean for the Example part ?

**Comment by frhadjmli** [comment]
@caupolicandiaz I made the changes in the review

**Review by caupolicandiaz — APPROVED** [review]
@frhadjmli Looks good. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@frhadjmli I did one minor format change, now it looks good to go.

**Comment by SSwiniarski** [comment]
@frhadjmli Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/go/math-functions/floor

image

**Pull request merged**